TCD-2903: Backtrace - Unity (#2153)
saucelabs/sauce-docs: docs/error-reporting/platform-integrations/unity/help-shift.md

Reviewer: Can you rename the file from "help-shift.md" to "helpshift.md"?

---

> id: help-shift

Reviewer: ```suggestion
id: helpshift
```

title: Unity Backtrace and HelpShift SDK integration
sidebar_label: HelpShift SDK

> description: Integrate HelpShift SDK in Unity games. 

Author: ```suggestion
description: Integrate HelpShift SDK in Unity games.
```

## 1. Integrating Both SDKs

Integrate the respective SDKs for Backtrace and HelpShift first.

### Backtrace SDK

- [Unity Integration Guide for Backtrace](https://support.backtrace.io/hc/en-us/articles/360040515991)

### HelpShift SDK

- iOS
  - [HelpShift for iOS](https://developers.helpshift.com/unity/getting-started-ios/)
- Android
  - [HelpShift for Android](https://developers.helpshift.com/unity/getting-started-android/)
- WebGL (web chat)
  - [Helpshift WebChat](https://developers.helpshift.com/web-chat/getting-started/)
  - Refer to the [Unity documentation](https://docs.unity3d.com/Manual/webgl-interactingwithbrowserscripting.html) on how to embed a JavaScript library.
  - Find the source code in our [example game](https://github.com/backtrace-labs/unity-asterax/blob/Helpshift-Webinar/Assets/Plugins/AsteraX.jslib)

## 2. Linking HelpShift And Backtrace

### iOS And Android

For details on controlling the display of the HelpShift UI, refer to the [HelpShift SDK for Android documentation](https://developers.helpshift.com/sdkx-unity/support-tools-android/#conversation-view).

> Linking Backtrace and HelpShift is easy. Simply add the `backtraceClient["guid"]` to a customer issue field with the ID `device_id`, as shown in the code example below:

Author: ```suggestion
Linking Backtrace and HelpShift is not tricky. Add the `backtraceClient["guid"]` to a customer issue field with the ID `device_id`, as shown in the code example below:
```
